### ISSUE TITLE
XPAwardTable incorrect CL 4\CR 5

### ISSUE THREAD

**Issue by anatoliy-savchak** [GrognardsFromHell/TemplePlus#662]
xpawarddd->XPAwardTable[4-1][5+2] has 1800 but XP awards for killing CR 5 should be 1600 at level 4.

**Comment by doug1234** [comment]
That does appear to be incorrect.  Have you seen any other values that are off?

**Comment by anatoliy-savchak** [comment]
> That does appear to be incorrect. Have you seen any other values that are off?

No. I decided that when if someone is going to fix that one number he will recheck them all. Even if that would be me))

**Comment by DudeMcDude** [comment]
I think the lower levels are special-cased. Haven't thoroughly checked though.
